### PR TITLE
Update test to check if path exists.

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: test/vc-examples.spec.js [hunk 1]
```diff
@@ -3,12 +3,17 @@
...
 const should = chai.should();
 
 describe('credentialsDir', () => {
   it('should exist path to credentials directory.', async () => {
     should.exist(credentialsDir);
     credentialsDir.should.be.a('string');
+
+    const exists = fs.existsSync(credentialsDir);
+
+    exists.should.equal(true);
```
Comment (davidlehn) [inline test/vc-examples.spec.js:17]: Could make this a one-liner:
```suggestion
    fs.existsSync(credentialsDir).should.be.true();
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+## 2.0.0 - 2023-02-TBD
 
 ### Added
 - Add test.
 - Add image for `jff`.
 
 ### Changed
 - Update `Usage` section in `README.md`.
-- Update names of `.json` files containing credentials to `credential.json`.
+- **Breaking**: Update names of `.json` files containing credentials to
```
Comment (davidlehn) [inline CHANGELOG.md:11]:
```suggestion
- **BREAKING**: Update names of `.json` files containing credentials to
```

Comment (JSAssassin) [reply CHANGELOG.md]: Fixed here [6f3a88e](https://github.com/credential-handler/vc-examples/pull/8/commits/6f3a88e20b55fa8f668753fa24b9a006f68ea4f2)

File: test/vc-examples.spec.js (hunk 1 above)
Comment (JSAssassin) [reply test/vc-examples.spec.js]: Updated [6f3a88e](https://github.com/credential-handler/vc-examples/pull/8/commits/6f3a88e20b55fa8f668753fa24b9a006f68ea4f2)

**Review by davidlehn — APPROVED** [review]

**Pull request merged**